### PR TITLE
PR for 175_EMT_GFM_VSC model

### PR DESCRIPTION
**Pull request by Carlos-Alegre**
Pull request for issue #175 EMT Grid Forming VSC model - colib project

### CODE REVIEW

File: .vscode/settings.json
Comment (GJCRESYM) [inline .vscode/settings.json:1]: Remove this file

File: pages/models/generations/Sources/VSC/EMTGridFormingVSC/index.md [hunk 1]
```diff
@@ -0,0 +1,354 @@
+---
+layout: page 
+title: Generic EMT Grid Forming Voltage Source Converter 
+tags: [Opensource, EMT, voltage source, converter, wind, pv, hdvc, dynawo, STEPSS, grid following]
```
Comment (GJCRESYM) [inline pages/models/generations/Sources/VSC/EMTGridFormingVSC/index.md:4]: Provide double quotes to tags

File: pages/models/generations/Sources/VSC/EMTGridFormingVSC/index.md [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+
+
+## Open source implementations
+
+This model has been successfully implemented in :
+
+| Software      | URL | Language | Open-Source License | Last consulted date | Comments |
+| PSTess | [Link](https://github.com/sandialabs/snl-pstess/blob/master/pstess/gfma.m) | MATLAB | Copyright 2021 National Technology & Engineering Solutions of Sandia, LLC
+(NTESS) | 09/07/2024 | Droop control version, equivalent to REGFM_A1 from WECC accepted dynamic models|
+
```
Comment (GJCRESYM) [inline pages/models/generations/Sources/VSC/EMTGridFormingVSC/index.md:333]: Values not populated correctly. Please self-review and update it

**Review by GJCRESYM — CHANGES_REQUESTED** [review]

File: .vscode/settings.json
Comment (Carlos-Alegre) [reply .vscode/settings.json:1]: removed

File: pages/models/generations/Sources/VSC/EMTGridFormingVSC/index.md (hunk 2 above)
Comment (Carlos-Alegre) [reply pages/models/generations/Sources/VSC/EMTGridFormingVSC/index.md:333]: corrected

**Review by GJCRESYM — APPROVED** [review]

File: pages/models/generations/Sources/VSC/EMTGridFormingVSC/index.md [hunk 3]
```diff
@@ -0,0 +1,354 @@
+---
+layout: page 
+title: Generic EMT Grid Forming Voltage Source Converter 
+tags: ["Opensource", "EMT", "voltage source", "converter", "wind", "pv", "hdvc", "Droop Control", "Virtual Synchronous Machine", "grid forming", "PSTess"]
```
Comment (GJCRESYM) [inline pages/models/generations/Sources/VSC/EMTGridFormingVSC/index.md:4]: typo: HVDC

Comment (GJCRESYM) [reply pages/models/generations/Sources/VSC/EMTGridFormingVSC/index.md]: resolved

**Pull request merged**